tests/extended/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+find_package(Boost REQUIRED COMPONENTS system)
+
+add_executable(test_extended dnsa_connection_pool.cpp ../main.cpp)
+target_link_libraries(test_extended Boost::system)
+setup_test(test_extended)
+target_compile_options(test_extended PRIVATE -Wno-deprecated-declarations)
+
+add_test(
+  NAME test_extended
+  COMMAND sudo ${CMAKE_CURRENT_SOURCE_DIR}/../runtest.sh
```

Comment:
The fact that our test requires sudo (which isn't noninteractive in general case) and modifies hosts file is really awkward. This deserves comment highlighting that - and perhaps TODO to run this test inside container.

Reply:
Maybe the extended tests should be off by default (and explicitly set on in CI run via -D flag)), I feel that was the intent behind separating them in a first place.

Reply:
Test is turned off, todo was added.